service/SlackService.ts

```diff
@@ -97,32 +58,27 @@ class SlackService {
     return filteredUsers;
   }
 
-  private getUsers(): Promise<ReadonlyArray<SlackUser>> {
-    const url = "https://slack.com/api/users.list?token=" + this.slackConfiguration.appToken;
+  public getUsers(): Promise<ReadonlyArray<SlackUser>> {
+    const url = "https://slack.com/api/users.list"
     if (this.slackConfiguration.dryRun) {
       return Promise.resolve([]);
     } else {
-      return request.get(url)
-        .then((response) => JSON.parse(response).members
-          .map((member) => new SlackUser(member.id, member.profile.real_name, member.profile.email)));
+      return axios.get(url, {
+        headers: {'Authorization': `Bearer ${this.slackConfiguration.appToken}`}
+      })
+        .then((response) => {
+          // TODO missing paging with cursors
```

Comment:
How much do you expect Nitor employee headcount to grow to warrant cursors? 💯

Reply:
Well, we already need to use paging with Personio API because number of employees.